template/template.go

```diff
@@ -20,6 +24,28 @@ func init() {
 // Render parses and executes a template, returning the results
 // in string format.
 func Render(template string, playload *drone.Payload) (string, error) {
+	if strings.HasPrefix(template, "http") {
+		parsedURL, err := url.Parse(template)
+
+		if err != nil {
+			return "", errors.New(fmt.Sprintln("Failed to parse template URL: ", err))
+		}
+
+		resp, err := http.Get(parsedURL.String())
+
+		if err != nil {
+			return "", errors.New(fmt.Sprintln("Failed to fetch remote template: ", err))
+		}
+
```

Comment:
needs a `defef resp.Body.Close()` ... the client is always responsible for closing the connection, and failure to do so will result in open filedscription buildup until the program quits. Of course this probably isn't a big deal for short-lived plugins, but still a good habit to get into :)